dotcom-rendering/src/components/Card/components/CardLayout.tsx

```diff
@@ -162,27 +164,30 @@ export const CardLayout = ({
 	minWidthInPixels,
 	imageType,
 	containerType,
-	gapSize = 'small',
+	gapSizes = { row: 'small', column: 'small' },
```

Comment:
Non-blocking as preexisting: it looks like there is only one usage of card layout and as Card always provides a gapsizes object,  we could make this type required and remove the default here.

Reply:
Nice spot!